packages/app-bridge/src/registries/commands/CommandRegistry.ts

```diff
@@ -20,4 +22,6 @@ export type CommandRegistry = CommandNameValidator<{
     downloadAsset: DownloadAsset;
     openNavigationManager: OpenNavigationManager;
     openNewPublication: OpenNewPublicationPayload;
+    openSearchDialog: OpenSearchDialog;
+    closeSearchDialog: CloseSearchDialog;
```

Comment:
Wouldn't this do the same job? Having a bunch of types which are basically an alias for `void` is just adding one more step for anyone who tries to read the code.

```suggestion
    openSearchDialog: void;
    closeSearchDialog: void;
```

Reply:
Thanks for your feedback. As we’re going to clean up the AppBridgeTheme very soon I followed the same pattern that we currently use for now.

Reply:
If following a pattern introduces more work for a cleanup that is already planned, I would prioritise reducing future workload over following a pattern we're planning to scrap. If you start breaking that pattern now, it will remind us to clean up the other occurrences when we start cleaning up the AppBridgeTheme, as well as give an idea how the result should (could) look like after the cleanup to whoever works on it.